src/share/classes/sun/misc/SharedSecrets.java

```diff
@@ -107,6 +108,14 @@ public static JavaNetAccess getJavaNetAccess() {
         return javaNetAccess;
     }
```

Comment:
can we reuse javaNetAccess for our purpose?

Reply:
JavaNetAccess is used inside URLClassLoader, which is calling the private method of URLClassLoader, I don’t want to change too much structure. So I added the Access class.

http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/net/URLClassLoader.java#l759